timelib.h

```diff
@@ -488,7 +566,7 @@ timelib_long timelib_parse_zone(char **ptr, int *dst, timelib_time *t, int *tz_n
  * inspect errors->warnings_count.
  */
 void timelib_strtointerval(char *s, size_t len,
-                           timelib_time **begin, timelib_time **end,
+						   timelib_time **begin, timelib_time **end,
 						   timelib_rel_time **period, int *recurrences,
 						   timelib_error_container **errors);
 
```

Comment:
If it was me, I would have changed the tabs for spaces here, as it's alignment, not indentation, but "meh".

Reply:
Changed to spaces.